src/includes/getting-started-install/java.mdx

```diff
@@ -13,3 +11,22 @@ dependencies {
     implementation 'io.sentry:sentry:3.1.0'
 }
 ```
+
+```xml {tabTitle:Maven}
```

Comment:
Is it possible to add a file name too? `pom.xml` is the file name here. I dunno about SBT though
```suggestion
```xml {tabTitle:Maven}
```

Reply:
I'll give it a go.